packages/block-library/src/comments-pagination-numbers/block.json

```diff
@@ -31,6 +31,22 @@
 				"fontSize": true
 			}
 		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"__experimentalBorder": {
```

Comment:
@rinkalpagdar  Thanks for the PR.

It might be best to hide these border controls by default.